spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'posts/show.html.erb', type: :view do
+  before(:each) do
+    @user = User.create(name: 'Davon',
+                        photo: 'https://picsum.photos/200',
+                        bio: 'Nigerian Chemist',
+                        posts_counter: 0)
+    @any_post = Post.create(author_id: @user.id, title: 'Hello', text: 'This is any post', comments_counter: 0,
+                            likes_counter: 0)
+    Comment.create(post: @any_post, author_id: @user.id, text: 'This is the 1st comment on the 4th post')
+    Comment.create(post: @any_post, author_id: @user.id, text: 'This is the 2nd comment on the 4th post')
+    Comment.create(post: @any_post, author_id: @user.id, text: 'This is the 3rd comment on the 4th post')
+    visit user_posts_path(@user, @any_post)
+  end
+
+  it 'I can see who wrote the post' do
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'shows number of comments a post has' do
+    expect(page.body).to include('Comments: 3')
+  end
+
+  it 'shows number of likes a post has' do
+    expect(page.body).to include('Likes: 0')
+  end
+
+  it 'shows the title of the post' do
+    expect(page).to have_content(@any_post.title)
+  end
+
+  it 'shows part of a post body (This is any post)' do
+    expect(page).to have_content('This is any post')
+  end
+
+  it 'shows the user name of a comment' do
+    expect(page).to have_content('Davon')
+  end
```

Comment:
- [ ] You have done well writing the test for the `User` **post index page**:`, but you're missing some of the required tests which  are important, I have pointed them out in the screenshot below, I'll suggest you look at the requirements and implement these. :+1: :+1: 

- - Post index requirements:
![Selection_072](https://github.com/Ademola-coding/Blog_rails/assets/107632780/4a1fe0df-0a95-44a4-95b3-68a00fc4a1c3)

- [ ] Also your describe has `describe 'posts/show.html.erb', ` but this is meant to be the post index test. I'll suggest you always make sure you are writing tests for the correct views. :wave: :small_airplane:

Reply:
![Screenshot (133)](https://github.com/Ademola-coding/Blog_rails/assets/96092850/494a84de-a0e1-464c-b91d-66a47b24628e)

made proper corrections